### PR TITLE
Add support for SVGs handled through SDWebImageSwiftUI

### PR DESCRIPTION
**Pull request by NgaiJustin**
## Overview

Problem: URLs that linked to images in SVG format were not display.
Solution: Add support for SVGs handled through SDWebImageSwiftUI

## Test Coverage

- Tested on iPhone XR

## Screenshots (delete if not applicable)

<details>

  <summary>Before</summary>

  Image from iOS
  

</details>

<details>

  <summary>After</summary>

  IMG_3091
  

</details>

### CODE REVIEW

**Review by Npikielny — APPROVED** [review]
Lgtm!

File: Volume/Supporting/AppDelegate.swift [hunk 1]
```diff
@@ -15,6 +16,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     private let notificationIntervalKey = "notificationIntervalKey"
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+        // Add SVG Support
+        SDImageCodersManager.shared.addCoder(SDImageSVGCoder.shared)
```
Comment (dishanest) [inline Volume/Supporting/AppDelegate.swift:20]: How does adding a line here enable SVG rendering on other screens? Just curious.

**Review by dishanest — APPROVED** [review]
Sick fix!

File: Volume/Supporting/AppDelegate.swift (hunk 1 above)
Comment (NgaiJustin) [reply Volume/Supporting/AppDelegate.swift:20]: We were using the SDWebImage framework for all the screens, which does not support vector images by default. By registering this additional coder on launch, we can extend the WebImage support. 
https://github.com/SDWebImage/SDWebImageSVGCoder

**Review by emarc0314 — APPROVED** [review]
lgtm 🔥

**Pull request merged**